### PR TITLE
Qa new

### PR DESCRIPTION
**Pull request by superpoussin22**
### Describe the Pull Request

```
Little Qa:
correct a shell script 
replace chmod using subprocess with os.chmod 
reduce rights
correct a typo
```

### Checklist for PR

- [X] Run MobSF unit tests and lint `tox -e lint,test`.
- [X] Tested Working on Linux, Mac, Windows, and Docker
- [X] Make sure build is passing on your PR. [Build Status](https://travis-ci.com/MobSF/Mobile-Security-Framework-MobSF/pull_requests)

### CODE REVIEW

File: MobSF/utils.py [hunk 1]
```diff
@@ -445,11 +445,11 @@ def get_adb():
             adb = 'adb'
             if platform.system() == 'Darwin':
                 adb_dir = os.path.join(settings.TOOLS_DIR, 'adb/mac/')
-                subprocess.call(['chmod', '777', adb_dir])
+                os.chmod(adb_dir, 0o744)
```
Comment (ajinabraham) [inline MobSF/utils.py]: 777 or 774 is enough?

File: MobSF/utils.py [hunk 2]
```diff
@@ -445,11 +445,11 @@ def get_adb():
...
                 adb = os.path.join(settings.TOOLS_DIR, 'adb/mac/adb')
             elif platform.system() == 'Linux':
                 adb_dir = os.path.join(settings.TOOLS_DIR, 'adb/linux/')
-                subprocess.call(['chmod', '777', adb_dir])
+                os.chmod(adb_dir, 0o744)
```
Comment (ajinabraham) [inline MobSF/utils.py]: Same here

File: StaticAnalyzer/views/ios/binary_analysis.py [hunk 3]
```diff
@@ -358,21 +358,21 @@ def class_dump(tools_dir, bin_path, app_dir, bin_type):
...
             else:
-                logger.info('Running class-dump-z aganst binary')
+                logger.info('Running class-dump-z against binary')
                 if (len(settings.CLASSDUMPZ_BINARY) > 0
                         and is_file_exists(settings.CLASSDUMPZ_BINARY)):
                     class_dump_bin = settings.CLASSDUMPZ_BINARY
                 else:
                     class_dump_bin = os.path.join(tools_dir, 'class-dump-z')
-            os.chmod(class_dump_bin, 0o777)
+            os.chmod(class_dump_bin, 0o744)
```
Comment (ajinabraham) [inline StaticAnalyzer/views/ios/binary_analysis.py]: Here too

File: StaticAnalyzer/views/ios/binary_analysis.py [hunk 4]
```diff
@@ -381,7 +381,7 @@ def class_dump(tools_dir, bin_path, app_dir, bin_type):
                 jtool_bin = settings.JTOOL_BINARY
             else:
                 jtool_bin = os.path.join(tools_dir, 'jtool.ELF64')
-            os.chmod(jtool_bin, 0o777)
+            os.chmod(jtool_bin, 0o744)
```
Comment (ajinabraham) [inline StaticAnalyzer/views/ios/binary_analysis.py]: Here

File: scripts/postgres_support.sh [hunk 5]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -e
 POSTGRES=$1
-echo 'Postgres support : ${POSTGRES}'
+echo "Postgres support : ${POSTGRES}"
 if [ "$POSTGRES" == True ]; then
```
Comment (ajinabraham) [inline scripts/postgres_support.sh]: Yeah this is the problem.. interpolation need "

File: MobSF/utils.py (hunk 1 above)
Comment (superpoussin22) [reply MobSF/utils.py]: in my test yes but it means that the user who do the install is the same who launch MobSF

File: MobSF/utils.py (hunk 2 above)
Comment (superpoussin22) [reply MobSF/utils.py]: in my test yes but it means that the user who do the install is the same who launch MobSF 
774 can be a safe alternative

File: StaticAnalyzer/views/ios/binary_analysis.py (hunk 3 above)
Comment (superpoussin22) [reply StaticAnalyzer/views/ios/binary_analysis.py]: in my test yes but it means that the user who do the install is the same who launch MobSF 
774 can be a safe alternative

File: StaticAnalyzer/views/ios/binary_analysis.py (hunk 4 above)
Comment (superpoussin22) [reply StaticAnalyzer/views/ios/binary_analysis.py]: in my test yes but it means that the user who do the install is the same who launch MobSF 
774 can be a safe alternative

**Pull request merged**